### PR TITLE
docs: installing update

### PR DESCRIPTION
**Pull request by bjlittle**
### Description
This PR replaces #4256, originally raised by @kant, but has sadly gone stale after no response for six months.

Seems like a low hanging fruit that we can easily bank in time for the `iris` 3.2.0 release.

Particularly keen to do this as the change is motivated by a community user wanting to improve the documentation.

The rendered change of this PR is available to view in the associated built documentation [available here](https://bjlittle-iris.readthedocs.io/en/latest/installing.html#installing-from-source-with-conda-developers). 

---
[Consult Iris pull request check list]( https://scitools-iris.readthedocs.io/en/latest/developers_guide/contributing_pull_request_checklist.html)

### CODE REVIEW

File: docs/src/installing.rst [hunk 1]
```diff
@@ -85,10 +85,12 @@ local copy of Iris::
...
-The ``--force`` option is used when creating the environment, this is optional
-and will force the any existing ``iris-dev`` conda environment to be deleted
-first if present.  This is useful when rebuilding your environment due to a
-change in requirements.
+.. note::
+
+  The ``--force`` option is used when creating the environment, but first
+  removes any previously existing ``iris-dev`` environment of the same name
+  beforehand.  This is particularly useful when rebuilding your environment
+  due to a change in requirements.
```
Comment (rcomer) [inline docs/src/installing.rst:93]:
```suggestion
  The ``--force`` option, used when creating the environment, first removes
  any previously existing ``iris-dev`` environment of the same name. This is
  particularly useful when rebuilding your environment due to a change in
  requirements.
```
This scans better in my head.  I think having both "first" and "beforehand" is redundant.

File: docs/src/installing.rst [hunk 2]
```diff
@@ -85,10 +85,12 @@ local copy of Iris::
...
 
-The ``--force`` option is used when creating the environment, this is optional
-and will force the any existing ``iris-dev`` conda environment to be deleted
-first if present.  This is useful when rebuilding your environment due to a
-change in requirements.
+.. note::
+
+  The ``--force`` option is used when creating the environment, but first
+  removes any previously existing ``iris-dev`` environment of the same name
+  beforehand.  This is particularly useful when rebuilding your environment
```
Comment (stephenworsley) [inline docs/src/installing.rst:92]: I feel like this use of "beforehand" is redundant given the use of "but first".

**Review by stephenworsley — CHANGES_REQUESTED** [review]
Looks good, just a small comment.

File: docs/src/installing.rst (hunk 2 above)
Comment (rcomer) [reply docs/src/installing.rst:92]: snap!

Comment (bjlittle) [reply docs/src/installing.rst]: Nice one, thanks guys :+1: 

Done

**Review by stephenworsley — APPROVED** [review]

**Comment by bjlittle** [comment]
Note that, the original PR #4256 by @kant would have most likely been banked straight away if we didn't have to jump through the hoop of the CLA :cry: 

Why do we need the CLA? It's such a barrier to community contributions such as #4256

For me, that's really a loss not a win.

**Comment by trexfeathers** [comment]
>Why do we need the CLA?

Is anyone progressing this at the moment? Do you we need to action a specific team member to speak to Met Office legal?

**Comment by bjlittle** [comment]
> > Why do we need the CLA?
> 
> Is anyone progressing this at the moment? Do you we need to action a specific team member to speak to Met Office legal?

I pinged them back in Nov 2021. Additionally, @lbdreyer is actively pursuing this now :star_struck: 

Also related to https://github.com/SciTools/cartopy/issues/1919

**Comment by bjlittle** [comment]
That said, [this](https://ben.balter.com/2018/01/02/why-you-probably-shouldnt-add-a-cla-to-your-open-source-project/) is an interesting read... although, I genuinely know nothing about such things. 

Hence the push to engage with MO legal and act on best advice.

**Review by rcomer — APPROVED** [review]

**Pull request merged**